chat/room.go

```diff
@@ -235,6 +235,14 @@ func (r *Room) NamesPrefix(prefix string) []string {
 		users = append(users, item.Value().(*Member).User)
 	}
 	sort.Sort(message.RecentActiveUsers(users))
+	for i, user := range users {
```

Comment:
Is it necessary to change the signature of a public method and break API compatibility?

Can we put this in `completeName` instead?